.goreleaser.yml

```diff
@@ -134,36 +137,154 @@ nfpms:
     formats:
       - rpm
       - deb
-    bindir: /opt/observiq-otel-collector
+    bindir: /usr/share/observiq-otel-collector/stage/observiq-otel-collector
```

Comment:
This matches the directory all other files are installed to. Post install script copies these files to /opt/observiq-otel-collector.